docs/state-sync.md

```diff
@@ -0,0 +1,168 @@
+# State Sync
+
+## What is State sync?
+
+The Provenance 1.10.0+ release comes with Tendermint Core 0.34 which includes support for state sync. State sync allows a new node to join a network by fetching a snapshot of the application state at a recent height vs. fetching and replaying all historical blocks (which can take days). This can reduce the time needed to sync with the network down to minutes.
+
+<!-- TOC -->
+- [What is State sync?](#what-is-state-sync)
+    - [Starting a new node from State Sync](#starting-a-new-node-from-state-sync)
+        - [Mainnet](#mainnet)
+        - [Testnet](#testnet)
+    - [Power users](#power-users)
+
+
+### Starting a new node from State Sync
```

Comment:
All of these heading levels after "What is State sync" should be reduced by one (and then the TOC should be regenerated).

Reply:
Actually, The `What is State sync?` heading should probably just be removed so that the paragraph under it is under `# State Sync`. Then, again, all headings (except the top  one) should be reduced by 1.

Reply:
👍